tests/queries/0_stateless/02902_diable_apply_deleted_mask.sql

```diff
@@ -0,0 +1,25 @@
+DROP TABLE IF EXISTS test_apply_deleted_mask;
+
+CREATE TABLE test_apply_deleted_mask(id Int64, value String) ENGINE = MergeTree ORDER BY id;
+
+INSERT INTO test_apply_deleted_mask SELECT number, number::String FROM numbers(5);
+
+DELETE FROM test_apply_deleted_mask WHERE id % 2 = 0;
+
+SELECT 'Normal SELECT does not see deleted rows';
+SELECT *, _row_exists FROM test_apply_deleted_mask;
+
+SELECT 'With the setting enabled the deleted rows are visible';
+SELECT *, _row_exists FROM test_apply_deleted_mask SETTINGS apply_deleted_mask = 0;
+
+SELECT 'With the setting enabled the deleted rows are visible but still can be filterd out';
```

Comment:
```suggestion
SELECT 'With the setting disabled the deleted rows are visible but still can be filterd out';
```
same here, it should be disabled right ?